lab-beeksma/lib/dataFromFile.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+
+function logSpecificFile(filename, cb) {
+  fs.readFile(`${__dirname}/../assets/${filename}`, function (err, res) {
+    if(err){
+      cb(err,undefined);
+    }
+    cb(err,res.slice(0,8).toString('hex'));
+  });
+}
+
+exports.logData = function(files, cb) {
+  var  loopCount = 0;
+  var results = [];
+  function getResults(err,res){
+    if(err){
+      cb(err, res);
+      return;
+    }
+    if(res){
```

Comment:
You don't need to check `res` here, since we know this isn't an error scenario (i.e. `res` is a valid, whatever it is).